### PR TITLE
[feat] Dockerization of the project

### PR DESCRIPTION
**Pull request by D4rkHeart**

### CODE REVIEW

**Comment by multiscan** [comment]
as sugested by @domq it would be great if while building the image you could run npm install too as this will make app startup much faster.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+
+#Update ...
+RUN apt-get update
+RUN apt-get upgrade -y
+#Install node Js
+RUN apt install nodejs -y
+#Intall npm
+RUN apt install npm -y
```
Comment (ponsfrilus) [inline Dockerfile]: `RUN apt install -y nodejs npm` ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+
+#Update ...
+RUN apt-get update
+RUN apt-get upgrade -y
```
Comment (ponsfrilus) [inline Dockerfile]: Is it really necessary ?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+#Install node Js
+RUN apt install nodejs -y
+#Intall npm
+RUN apt install npm -y
+#Create a folder and create directory  
+RUN mkdir /botpresence 
+WORKDIR  /botpresence
```
Comment (ponsfrilus) [inline Dockerfile]: Double spaces

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+run :
+Docker-compose up -- build
```
Comment (ponsfrilus) [inline Makefile]: No capital D

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+version: "3.3"  # optional since v1.27.0
+services:
+  botpresence:
+    image : botpresence:latest
```
Comment (ponsfrilus) [inline docker-compose.yml]: How can I get or create this image ?

**Review by ponsfrilus — CHANGES_REQUESTED** [review]
Please look at my comments.

**Comment by D4rkHeart** [comment]
> as sugested by @domq it would be great if while building the image you could run npm install too as this will make app startup much faster.

Ho thanks for the reminder i'll do it right now.

File: Dockerfile (hunk 3 above)
Comment (D4rkHeart) [reply Dockerfile]: thanks i made the correction.

File: Makefile (hunk 4 above)
Comment (D4rkHeart) [reply Makefile]: thanks i made the correction ! ^^

File: Dockerfile (hunk 2 above)
Comment (D4rkHeart) [reply Dockerfile]: To have the latest version of everything in the bot because of its interaction with Telegram which is continuously updated no ?

Comment (ponsfrilus) [reply Dockerfile]: Isn't that the job of the docker image ([https://hub.docker.com/_/ubuntu](https://hub.docker.com/_/ubuntu)), e.g. with `docker pull ubuntu:20.04`?

Comment (D4rkHeart) [reply Dockerfile]: you right ! they are useless.

**Review by ponsfrilus — APPROVED** [review]
LGTM

**Pull request merged**